deployment/terraform/aws/validator/main.tf

```diff
@@ -30,16 +30,28 @@ resource "aws_instance" "this_node" {
   key_name   = aws_key_pair.key_pair.id
   monitoring = true
 
+  connection {
+    type        = "ssh"
+    host        = self.public_ip
+    user        = var.ssh_username
+    private_key = file(var.ssh_private_key_path)
+  }
+
+  provisioner "remote-exec" {
+    script = "./provisioner/install-ansible-deps.sh"
+  }
+
   tags = {
     Name = "Validator Node"
   }
 
   root_block_device {
     encrypted   = true
-    volume_size = 30
+    volume_size = 80
```

Comment:
```suggestion
    volume_size = 80
    delete_on_termination = true
```

Reply:
`delete_on_termination` defaults to `true` according to [terraform docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#delete_on_termination)